.github/workflows/eden_rpi.yml

```diff
@@ -0,0 +1,135 @@
+---
+name: EdenRPi4
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - "master"
+      - "[0-9]+.[0-9]+"
```

Comment:
@rvs Will we test everything that is pushed to master and all releases? I think that will suit us.